src/main/java/seedu/address/logic/Messages.java

```diff
@@ -40,11 +40,11 @@ public static String format(Person person) {
                 .append("; Phone: ")
                 .append(person.getPhone())
                 .append("; Email: ")
-                .append(person.getEmail())
-                .append("; Address: ")
-                .append(person.getAddress())
-                .append("; Tags: ");
-        person.getTags().forEach(builder::append);
+                .append(person.getEmail());
+        person.getAddress().ifPresent(address -> builder.append("; Address: ")
+                .append(person.getAddress().map(Object::toString).orElse(null)));
```

Comment:
I think it is better to change `.orElse(null)` to `orElse("")`, because passing null to `append()` method of `StringBuilder` would trigger the following error
```
Error:
|  reference to append is ambiguous
|    both method append(java.lang.CharSequence) in java.lang.StringBuilder and method append(char[]) in java.lang.StringBuilder match
```
Hence crashing the app. As a result, it is better to replace `null` with `""` to avoid ambiguity.

Reply:
Ok, after further testing I realized that if the null is passed in as the result of orElse, it is equivalent to `StringBuilder.append((String) null)` which does not crash the app but instead will literally append null to the end of the String, which may not be desired behaviour of our program, so may still be better to fix it